functions/sum/1.0/index.js

```diff
@@ -0,0 +1,21 @@
+const sum = async ({ left, operator, right }) => {
+  if (operator === "+") {
+    return {
+      result: left + right,
+    };
+  } else if (operator === "-") {
+    return {
+      result: left - right,
+    };
+  } else if (operator === "/") {
+    return {
+      result: left / right,
+    };
+  } else {
+    return {
+      result: left * right,
+    };
+  }
```

Comment:
Perhaps we could move the `return` to the end of the function en declare a `result` variable. Something like
```
let result;

if (operator === "+") {
 result = left + right;
} ... 
else {
  result = left * right;
}

return {
  result,
};
```